projects/aca-content/src/lib/components/info-drawer/metadata-tab/metadata-tab.component.ts

```diff
@@ -57,6 +59,14 @@ export class MetadataTabComponent implements OnInit, OnDestroy {
   @Input()
   node: Node;
 
+  /** Display tags in the card **/
+  @Input()
+  displayTags = true;
```

Comment:
Do we need this as property and Input? Looks like it is always true as you never pass to MetadataTabComponent something different than true (it is just hardcoded here and marked as Input even if is is never used as input). So you can just have in html: [displayTags]="true" and that will be enough instead of creating extra unused inputs. 

The same for displayCategories

Reply:
Updated!